src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -592,14 +598,9 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             textContent: {
-                condition: el => domUtils.isScriptElement(el) || domUtils.isStyleElement(el),
-
-                get: el => {
-                    if (domUtils.isScriptElement(el))
-                        return removeProcessingHeader(el.textContent);
+                condition: el => domUtils.isElementNode(el),
```

Comment:
`textContent` returns immediate child text nodes of element. We are interested only in `<script>` and `<style>` element text content, therefore new condition is incorrect.